src/status_im/contexts/wallet/events.cljs

```diff
@@ -485,3 +485,15 @@
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
                 :duration 10000}]]])})))
+
+(def send-asset-flow-config
```

Comment:
imo this should not be in the events file.

I think a better location for this is:
src/status_im/contexts/wallet/send/flow-config.cljs

Reply:
How about a namespace to store different flow configs?

Reply:
sure, one or the other. my only thing is that the flow-config should live close to where their data model is used given that the `skip-step?` functionality is coupled to that.

Reply:
Nice 👍